index.tsx

```diff
@@ -1322,7 +1328,7 @@ export function compiler(
 
     while (
       typeof arr[arr.length - 1] === 'string' &&
-      !arr[arr.length - 1].trim()
+      !(arr[arr.length - 1] as string).trim()
```

Comment:
was needed now that arr is no longer any. Seems like the check of string doesn't persist